keyboard.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>||Working with elements||</title>
+  <style>
+    #keyboard {
+      display: flex;
+    }
+    .white-key {
+      height: 100px;
+      width: 28px;
+      background-color: white;
+      outline-style: solid;
+      outline-width: thin
+    }
+
+    .black-key {
+      height: 60px;
+      background-color: black;
+      position: absolute;
+      outline-style: solid;
+      outline-width: thin
+    }
+  </style>
+</head>
+<body>
+  <div id="description"></div>
+  <div id="keyboard"></div>
+  <script> 
+    
+
+    document.body.onload = addWhiteKeys(14);
+    window.addEventListener('resize', resizeAllBlackKeys)
+    const allBlackKeysArray = Array.from(document.getElementsByClassName('black-key'))
```

Comment:
this could become outdated as you resize etc. to keep in in sync you could make it a function
```suggestion
    const getAllBlackKeysArray = () => Array.from(document.getElementsByClassName('black-key'))
```